docs/clients/index.md

```
@@ -0,0 +1,14 @@
+# Clients
+
+Django Language Server works with any client that supports the Language Server Protocol (LSP).
+
+### Adding a new client
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)